aospy/test/test_data_loader.py

```diff
@@ -788,5 +788,43 @@ def test_recursively_compute_variable_multi_level(load_variable_data_loader):
     np.testing.assert_array_equal(result.values, expected.values)
 
 
+def test_recursively_compute_grid_attr(load_variable_data_loader):
+    result = load_variable_data_loader.recursively_compute_variable(
+        bk, DatetimeNoLeap(5, 1, 1),
+        DatetimeNoLeap(5, 12, 31), model=example_model,
+        intvl_in='monthly')
+    filepath = os.path.join(os.path.split(ROOT_PATH)[0], 'netcdf',
+                            '00060101.sphum_monthly.nc')
+    expected = _open_ds_catch_warnings(filepath)['bk']
+    np.testing.assert_array_equal(result.values, expected.values)
+
+
+def test_recursively_compute_grid_attr_multi_level(load_variable_data_loader):
+    one_level = Var(
+        name='one_level', variables=(bk, ),
+        func=lambda x: 2 * x)
+    multi_level = Var(
+        name='multi_level', variables=(one_level, bk),
+        func=lambda x, y: x + y)
+    result = load_variable_data_loader.recursively_compute_variable(
+        multi_level, DatetimeNoLeap(5, 1, 1),
+        DatetimeNoLeap(5, 12, 31), model=example_model,
+        intvl_in='monthly')
+    filepath = os.path.join(os.path.split(ROOT_PATH)[0], 'netcdf',
+                            '00060101.sphum_monthly.nc')
+    expected = 3 * _open_ds_catch_warnings(filepath)['bk']
+    np.testing.assert_array_equal(result.values, expected.values)
+
+
+def test_recursively_compute_grid_attr_error(load_variable_data_loader):
+    zsurf = Var(name=ZSURF_STR, def_time=False, def_vert=False,
```

Comment:
Could you add a one-line comment explaining why this should fail?  Not self evident otherwise.